grakn-engine/src/main/java/ai/grakn/engine/postprocessing/UpdatingInstanceCountTask.java

```diff
@@ -55,7 +55,7 @@
 
     @Override
     public boolean start() {
-        final long shardingThreshold = engineConfiguration().getPropertyAsLong(AbstractGraknGraph.SHARDING_THRESHOLD);
+        final long shardingThreshold = engineConfiguration().getPropertyAsLong(GraknTxAbstract.SHARDING_THRESHOLD);
```

Comment:
Why the decision to flip the names? I feel like I prefer the old way, but if there's precedent for doing it like this, I don't mind.